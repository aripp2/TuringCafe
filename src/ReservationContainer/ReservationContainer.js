import React from 'react';
import Reservation from '../Reservation/Reservation';
import './ReservationContainer.scss';

const ReservationContainer = ({ reservations }) => {

  const allResys = reservations.map(resy => {
    return <Reservation 
      key={resy.id}
      id={resy.id}
      name={resy.name}
      date={resy.date}
      time={resy.time}
      number={resy.number}
    />
  })

  return (
      <section className='resy-container'>
          { allResys }
      </section>
    )
}

export default ReservationContainer;