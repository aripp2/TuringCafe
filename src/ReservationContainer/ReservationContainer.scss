.resy-container {
  width: 100%;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
  margin-top: 25px;
}