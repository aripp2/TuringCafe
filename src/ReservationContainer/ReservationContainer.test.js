import React from 'react';
import { shallow } from 'enzyme';
import ReservationContainer from './ReservationContainer';

describe('ReservationContainer', () => {
  let wrapper;
  const mockReservations = [
    {
        "id": 1,
        "name": "Christie",
        "date": "12/29",
        "time": "7:00",
        "number": 12
    },
    {
        "id": 2,
        "name": "Leta",
        "date": "4/5",
        "time": "7:00",
        "number": 2
    }
  ]

  beforeEach(() => {
    wrapper = shallow(<ReservationContainer
      reservations={mockReservations}
      />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})