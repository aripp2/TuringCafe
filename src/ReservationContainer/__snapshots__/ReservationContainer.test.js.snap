// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationContainer should match the snapshot 1`] = `
<section
  className="resy-container"
>
  <Reservation
    date="12/29"
    id={1}
    key="1"
    name="Christie"
    number={12}
    time="7:00"
  />
  <Reservation
    date="4/5"
    id={2}
    key="2"
    name="Leta"
    number={2}
    time="7:00"
  />
</section>
`;
